### ISSUE TITLE
Support for the VoiceOver screen reader on macOS.

### ISSUE THREAD

**Issue by diegov12** [wxMaxima-developers/wxmaxima#1613]
As of yet, I am unable to interact with wxmaxima using my screen reader. All menu items are accessible, but cannot access input/output.

 wxMaxima version: 21.11.0 MAC
 using w×Widgets version: wxWidgets 3.1.5
 Maxima version: 5.45.1
 Maxima build date: 2021-11-03 22:52:41
 Host type: arm-apple-darwin21.1.0
 System type: Darwin 21.3.0 ARM64
 Lisp implementation type: SBCl
 wxMaxima's idea of the directory layout is:
 User configuration dir: /Users/diego/.maxima/
 Help dir: /usr/local/Cellar/wxmaxima/21.11.0/wxMaxima.app/Contents/Resources

**Comment by daute** [comment]
Hi,
that seems to be an issue with wxwidgets, the framework we use.

But is a **command line program** (e.g. command line Maxima (with `display2d:false` set) not better suited for using a CAS with a screen reader?
E.g. something like (here I used the first tool (espeak), I found):

`
maxima --very-quiet --run-string="display2d:false$" | espeak --punct
`

wxMaxima tries to render formulas "as real as possible" (it renders fractions, integral signs, square root signs, ...), and I think a screen reader might not be able to describe that.

Best regards, Wolfgang

**Comment by diegov12** [comment]
Hello…

A big disadvantage to using a command line is the lack of keyboard shortcuts, which makes things a lot easier, even for editing code. Also note that some screen readers may not support terminal navigation, meaning that going back to check previous output becomes difficult. As for the display style, is TeX not an option with an the app? I remember coming across it in the maxima menu.
Thank you.

Diego Vicioso.

Email… ***@***.***

**Comment by gunterkoenigsmann** [comment]
WxMaxima now supports screen readers. But I believe wxWidgets still can only forward from wxMaxima to the operating system on MS windows, only. Hope one day somebody will write Screen reader support for wxWidgets for other operating systems, as well.
